crates/re_viewer_context/src/command_sender.rs

```diff
@@ -33,6 +33,10 @@ pub enum SystemCommand {
     /// is both modified and changed in the same frame.
     UpdateBlueprint(StoreId, Vec<DataRow>),
 
+    /// Show a timeline of the blueprint data.
+    #[cfg(debug_assertions)]
+    ShowBlueprintTimeline(bool),
```

Comment:
I find this misleading. It sounds like it opens a new panel with the time-stream of the blueprint data, but what it actually does is switch what store the streams panel shows.

I think this would be more clear if the `bool` was changed to a `stream_panel_store: StoreKind`, and this to

```suggestion
    /// Select what store is shown in the streams panel: the recording or the blueprint.
    #[cfg(debug_assertions)]
    SetActiveStreamPanelStore(StoreKind),
```